lang_interface/java/com/intel/daal/utils/LibUtils.java

```diff
@@ -48,24 +51,40 @@
      */
     public static void loadLibrary()
     {
-        try {
-            logger.log(logLevel, "Loading library " + DAALLIB + " as file.");
-            System.loadLibrary(DAALLIB);
-            logger.log(logLevel, "DONE: Loading library " + DAALLIB + " as file.");
+        if(loaded){
             return;
         }
-        catch (UnsatisfiedLinkError e) {
-            logger.log(logLevel, "Can`t find library " + DAALLIB + " in java.library.path.");
-        }
+        synchronized(LibUtils.class){
+            if(loaded){
+                return;
+            }
+            String v = System.getProperty("tbb-threads", "");
+            int numThreads = v.length()>0 ? Integer.valueOf(v):Runtime.getRuntime().availableProcessors();
+            logger.log("tbb-threads: " + numThreads);
```

Comment:
There is build failure associated with this line

lang_interface/java/com/intel/daal/utils/LibUtils.java:63: error: incompatible types: String cannot be converted to LogRecord
            logger.log("tbb-threads: " + numThreads);

Reply:
fixed. thank you.